tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -262,7 +262,7 @@ RUN SMDATAPARALLEL_TF=1 pip install --no-cache-dir ${SMDATAPARALLEL_BINARY}
 
 ENV LD_LIBRARY_PATH="/usr/local/lib/python3.7/site-packages/smdistributed/dataparallel/lib:$LD_LIBRARY_PATH"
 
-ADD https://raw.githubusercontent.com/aws/deep-learning-containers/master/src/deep_learning_container.py /usr/local/bin/deep_learning_container.py
+COPY deep_learning_container.py /usr/local/bin/deep_learning_container.py
 
 RUN chmod +x /usr/local/bin/deep_learning_container.py
```

Comment:
when is `/usr/local/bin/deep_learning_container.py` executed?

Reply:
It is executed by `import <framework>` Python statements within a container.

Reply:
ok